avocado/plugins/jobs.py

```diff
@@ -58,21 +58,17 @@ def _print_job_tests(self, tests):
             status = test.get('status')
             decorator = output.TEST_STATUS_DECORATOR_MAPPING.get(status)
             end = datetime.fromtimestamp(test.get('end'))
-            test_matrix.append((decorator(status, ''),
+            test_matrix.append((test.get('id'),
                                 end.strftime(date_fmt),
                                 "%5f" % float(test.get('time')),
-                                test.get('id')))
-        header = (output.TERM_SUPPORT.header_str('Status'),
```

Comment:
A few changes on this commit here were added on top of previous commits. So basically, you added and then changed. Shouldn't this be part of previous commits?